champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/passives/Longshot.java

```diff
@@ -123,7 +125,7 @@ public void onDamage(CustomDamageEvent event) {
         double length = UtilMath.offset(loc, event.getDamagee().getLocation());
         double damage = Math.min(getMaxDamage(level), length * getDamage(level));
 
-        event.setDamage(event.getDamage() + (damage));
+        event.setDamage(minDamage + (damage));
```

Comment:
This can potentially conflict with other skills, I propose changing the EventPriority to LOWEST

Reply:
meant LOW, not LOWEST